Library/Homebrew/install.rb

```diff
@@ -324,11 +337,27 @@ def install_formulae(
     def install_formula(formula_installer)
       f = formula_installer.formula
 
+      if formula_installer.dry_run?
+        print_dry_run_dependencies(f, formula_installer.compute_dependencies)
```

Comment:
Is this how this is handled for `upgrade`? Would be nice to consolidate some of that logic.

Reply:
> Is this how this is handled for upgrade? 

Yes, I made this using  `Upgrade#print_dry_run_dependencies`  as a model.
https://github.com/Homebrew/brew/blob/d4e6925cbb0e59e12c2a7862ae0c76c667232869/Library/Homebrew/upgrade.rb#L97